src/mappings/mappingHandlers.ts

```diff
@@ -1,7 +1,33 @@
 import { CosmosEvent, CosmosMessage } from "@subql/types-cosmos";
-import { Event, EvmLog, EvmTransaction, Message } from "../types";
+import { Event, EvmLog, EvmTransaction, Message, ChainAliases } from "../types";
 import { inputToFunctionSighash, isSuccess, isZero, stripObjectUnicode } from "../utils";
 
+const evmChainId: Record<string, string> = {
+    'cronosmainnet_25-1': '25',
+    'cronostestnet_338-1':'338'
+}
+
+let checkedAliases = false;
+
+async function setAliases() {
+    if (checkedAliases) return;
+
+    const cosmosChainId = await (global as any).api.getChainId();
+    if(!evmChainId[cosmosChainId]) {
+        checkedAliases = true;
+        return;
+    }
+
+    const chianAliases = ChainAliases.create({
+        id: 'chainId',
```

Comment:
```suggestion
        id: 'evmChainId',
```

We might want to add other aliases in the future. There will need to be an update in the other PR